Troubleshooting.md

```diff
@@ -324,6 +326,8 @@ Note: For newer Gson versions these rules might be applied automatically; make s
 }
 ```
 
+You can also use `<init>(...);` to keep all constructors of that class, but then you might actually rely on JDK Unsafe to create classes without no-args constructor, see [`GsonBuilder.disableJdkUnsafe()`](https://javadoc.io/doc/com.google.code.gson/gson/latest/com.google.gson/com/google/gson/GsonBuilder.html#disableJdkUnsafe()) for more information.

```

Comment:
Maybe change "rely on JDK Unsafe" to "rely on `sun.misc.Unsafe` on both JDK and Android". [`UnsafeAllocator.create()`](
https://github.com/google/gson/blob/8c523b8d2c2efe22a670fc63e5369692342f0632/gson/src/main/java/com/google/gson/internal/UnsafeAllocator.java#L47) also try some Android specific magic on older Dalvik VMs, but they are becoming quite rare, so I don't think it is needed to mention that.